### PR TITLE
Feature/ranking page list items/35

### PR DESCRIPTION
**Pull request by Yin-Lynn-Htun**
Rank list item for ranking page.
I commited the wrong file ( _RankingPage.js_ instead of _RankingListItem.module.css_ )  for 0e9e572b010b420e66104649ea4efa0e029b7f20.

### CODE REVIEW

**Review by hatoriMain — CHANGES_REQUESTED** [review]
Great work! The UI seems perfect!
Here are some tiny changes that may let our UI looks better and fit with the Figma ones.
I commented them directly on `Files changed`.
1. Height of the container.
2. Linear gradient stroke for each user's icon.

And thank you again for your hard work😊

File: src/pages/RankingPage/RankingListItem.module.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    text-shadow: 0px 4px 20px rgba(0, 0, 0, 0.25);
+}
+.avatar {
+    width: 7.5rem;
+    height: 7.5rem;
+    border-radius: 50%;
+    object-fit: cover;
+    margin: 0px 1.5rem;
+}
```
[the remark this answers is not in the document]
Comment (Yin-Lynn-Htun) [reply src/pages/RankingPage/RankingListItem.module.css:62]: This is so cool. I didn't know that CSS trick.

**Comment by hatoriMain** [comment]
The UI seems work very well! Thank you for your contribution!

**Comment by JiahuiChen99** [comment]
Remember to close the branch once the branch is merged 🙇‍♀️

**Pull request merged**